common/logisticspipes/utils/gui/ItemDisplay.java

```diff
@@ -271,9 +284,12 @@ public void renderItemArea(double zLevel) {
 			buf.begin(GL11.GL_QUADS, DefaultVertexFormats.POSITION_TEX);
 			int xPosition = (width / 2) - 50;
 			int yPosition = 40;
-			buf.pos(xPosition, yPosition + 100, zLevel).tex(0.04, 0.72 + (graphic * 0.03125)).endVertex();;
-			buf.pos(xPosition + 100, yPosition + 100, zLevel).tex(0.08, 0.72 + (graphic * 0.03125)).endVertex();;
-			buf.pos(xPosition + 100, yPosition, zLevel).tex(0.08, 0.69 + (graphic * 0.03125)).endVertex();;
+			buf.pos(xPosition, yPosition + 100, zLevel).tex(0.04, 0.72 + (graphic * 0.03125)).endVertex();
+			;
```

Comment:
Instead of creating new lines here, just remove them please.

Reply:
I don't really know what happened here but I'm guessing it was the auto formatting

Reply:
But yes Ill remove the semicolons